src/pages/dashboard.tsx

```diff
@@ -583,6 +583,8 @@ const Dashboard: React.FC<DashboardProps> = () => {
                             >
                               <CircularProgressbar
                                 value={currentAttendance?.present_percentage}
+                                background
```

Comment:
@suvarnakale no need to assign any value to background?

Reply:
No.. there is no need, it just a attribute, value already added in style section.